launcher/sdw-launcher.py

```diff
@@ -1,42 +1,33 @@
 #!/usr/bin/env python3
-from logging.handlers import TimedRotatingFileHandler
-from PyQt4 import QtGui
+from sdw_updater_gui.UpdaterApp import launch_securedrop_client
 from sdw_updater_gui.UpdaterApp import UpdaterApp
+from sdw_updater_gui.Updater import should_launch_updater
+from PyQt4 import QtGui
+from logging.handlers import TimedRotatingFileHandler
 import logging
 import os
 import sys
+import argparse
 
 DEFAULT_HOME = os.path.join(os.path.expanduser("~"), ".securedrop_launcher")
-logger = ""
```

Comment:
Just a note that this logging code is refactored into `sdw_util` in #445. No strong preference on sequencing though; I'll try to address remaining comments on #445 today so perhaps we can aim to land that one first.